WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -70,7 +71,7 @@ private set
 
 		private Task<HttpResponseMessage> ClearnetRequestAsync(HttpRequestMessage request, bool isolateStream, CancellationToken cancellationToken = default)
 		{
-			return new ClearnetHttpClient(DestinationUriAction).SendAsync(request, isolateStream, cancellationToken);
+			return new ClearnetHttpClient(WasabiClientFactory.HttpClient, DestinationUriAction).SendAsync(request, isolateStream, cancellationToken);
```

Comment:
Why not ClearnetHttpClient get this automatically in a static way?

Reply:
If I put it inside, you can't change it in any way. Then what typically happens is that people need to change the inner HttpClient anyway, so they pass a new argument and it gets messier and messier. 

The plan is to get `ClearnetRequestAsync` out of `TorHttpClient` as it is in the Tor parallelization PR. 

Anyway, you can always remove the first parameter of `ClearnetHttpClient` and access it statically, the point is that it's much harder the other way as it typically needs to bubble it up so it often never happens.

Reply:
We agreed to not use static, so this is fine. https://github.com/zkSNACKs/WalletWasabi/issues/5156#issuecomment-775199012

Reply:
Good. Unfortunately, master diverged quite a lot so it looks like it will be easier to do that again than trying to solve the conflicts. But then, it should be good.